components/Home/Bonding/BondingList/useBondingList.jsx

```diff
@@ -393,9 +393,7 @@ const useAddProjectChangeToProducts = () =>
   useCallback(
     (productList) =>
       productList.map((record) => {
-        // current price of the LP token is multiplied by 2
-        // because the price is for 1 LP token and
-        // we need the price for 2 LP tokens
+        // To calculate the price of LP we need to multiply the price by 2
```

Comment:
// To calculate the price of LP we need to multiply olasReserve/TotalSupply by 2